tests/src/JIT/opt/Devirtualization/enum.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+enum MyEnum { One, Two, Three, Four, Five, Six, Seven, Eight, Nine, Ten };
+
+// ToString should be devirtualizable
+
+public class Test
+{
+    public static int Main()
+    {
+        string s = (MyEnum.Seven).ToString();
+        return (s.Length == "Seven".Length ? 100 : -1);
```

Comment:
I think my comment may have been confusing (or I'm just not understanding your reply).  I was just wondering why you predicated reporting success on getting the length right, instead of the stronger condition of getting the string right.  Not a big concern, as I don't really expect we'll have a bug where we accidentally devirtualize/inline `(MyEnum.Seven).ToString()` into something that returns "Three" or "Eight", but I was just curious if there was a reason you'd written it this way.